test/e2e/deployimage/plugin_cluster_test.go

```diff
@@ -264,6 +264,23 @@ func Run(kbc *utils.TestContext, imageCR string) {
 		return err
 	}, time.Minute, time.Second).Should(Succeed())
 
+	//Testing the finalizer
+	EventuallyWithOffset(1, func() error {
+		_, err = kbc.Kubectl.Delete(true, "-f", sampleFilePath)
+		return err
+	}, time.Minute, time.Second).Should(Succeed())
+
+	EventuallyWithOffset(1, func() error {
+		events, err := kbc.Kubectl.Get(true, "events", "--field-selector", "type", "Warning",
```

Comment:
```suggestion
		events, err := kbc.Kubectl.Get(true, "events", "--field-selector=type=Warning",
```
TO fix the command:
<img width="1253" alt="Screenshot 2022-07-05 at 23 03 18" src="https://user-images.githubusercontent.com/7708031/177424680-ef95b454-3f44-45b0-9834-c4daaa8bb2d6.png">